### PR TITLE
ASoC: SOF: Intel: pci-tgl: add ADL-PS support

### PR DESCRIPTION
**Pull request by reddymur**
Add PCI DID for Intel Alder Lake PS.
Signed-off-by: Muralidhar Reddy <muralidhar.reddy@intel.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by bardliao** [comment]
test this please

File: sound/soc/sof/intel/pci-tgl.c [hunk 1]
```diff
@@ -176,6 +176,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 		.driver_data = (unsigned long)&adl_desc},
 	{ PCI_DEVICE(0x8086, 0x51cd), /* ADL-P */
 		.driver_data = (unsigned long)&adl_desc},
+	{ PCI_DEVICE(0x8086, 0x51c9), /* ADL-PS */
+		.driver_data = (unsigned long)&adl_desc},
```
Comment (kv2019i) [inline sound/soc/sof/intel/pci-tgl.c:180]: ~~Please add this between 51c8 and 51cd entries, so we keep numerical order.~~

File: sound/soc/sof/intel/pci-tgl.c [hunk 2]
```diff
@@ -176,6 +176,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 		.driver_data = (unsigned long)&adl_desc},
```
Comment (kv2019i) [inline sound/soc/sof/intel/pci-tgl.c:176]: Please add a an empty line before the signed-off-by line in the git commit message.

**Review by plbossart — APPROVED** [review]

**Review by ranj063 — APPROVED** [review]

**Review by kv2019i — COMMENTED** [review]
@plbossart @ranj063 uh, I had made comment on the day this PR was posted, but somehow the submit fail. I'll sumit them now. I can make a fixup to this.
UPDATE: on a second thought, let's keep this order. :) no fixup coming

**Pull request merged**